tasks/eth/021-holocene-protocol-versions/SignFromJson.s.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {Proxy} from "@eth-optimism-bedrock/src/universal/Proxy.sol";
+import {SystemConfig} from "@eth-optimism-bedrock/src/L1/SystemConfig.sol";
+import {ProtocolVersions, ProtocolVersion} from "@eth-optimism-bedrock/src/L1/ProtocolVersions.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
+
+contract SignFromJson is OriginalSignFromJson {
+    using LibString for string;
+
+    // Chains for this task.
+    string constant l1ChainName = "mainnet";
+    string constant l2ChainName = "op";
+
+    uint256 constant protoVerHolocene = 0x0000000000000000000000000000000000000009000000000000000000000000;
+
+    // Safe contract for this task.
+    address foundationUpgradesSafe = vm.envAddress("FOUNDATION_SAFE");
+
+    // All L1 proxy addresses.
+    Types.ContractSet proxies;
+
+    function setUp() public {
+        proxies = _getContractSet();
+    }
+
+    function getAllowedStorageAccess() internal view override returns (address[] memory allowed) {
+        allowed = new address[](2);
+        allowed[0] = foundationUpgradesSafe;
+        allowed[1] = proxies.ProtocolVersions;
+    }
```

Comment:
Could be dealt with by a new, simpler `SingleSign` module that is a single version of the `CouncilFoundationNestedSign` module.

Reply:
After our discussion do you think this is still worth doing on this PR @sebastianst ?